src/webview/columns/data-column.tsx

```diff
@@ -41,30 +41,30 @@ export class DataColumn implements ColumnContribution {
 
     protected renderGroups(range: BigIntMemoryRange, memory: Memory, options: TableRenderOptions): React.ReactNode {
         const groups = [];
-        let words: React.ReactNode[] = [];
+        let maus: React.ReactNode[] = [];
```

Comment:
'Mau' made me think of a cat noise, but here we have a Maus. :-)

Reply:
Yeah, I know that feeling. In German it is even worse, cats say "Miau" (close), but a "Maus" is a mouse, literally. :-)

Reply:
Indeed we had a discussion if to go for camel-case or not. ;-)

Reply:
And I think there is a `BigMaus` somewhere as well. :-)